assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -51,6 +62,60 @@
 		}
 	}
 
+	.mdc-button__label {
+		font-weight: $fw-primary-normal;
+	}
+
+	.googlesitekit-date-range-selector {
```

Comment:
Since I've simplified the styles and removed all styles from the mdc selector styles all of the changes are specific changes to match the native elements to the Figma designs specifically for the header. Let me know if you still feel this file needs to be split.